src/h5web/metadata-viewer/utils.ts

```diff
@@ -24,32 +24,22 @@ export function renderShape(shape: Shape): string {
 }
 
 export function renderType(type: HDF5Type): string {
-  // Remove leading `H5T_`
-  const classLabel = type.class.slice(4).toLowerCase();
-
-  if (
-    type.class === HDF5TypeClass.Integer ||
-    type.class === HDF5TypeClass.Float
-  ) {
+  if (isNumericType(type)) {
     const { endianness, size } = type;
-    const typeInfos = `${classLabel}, ${
-      type.class === HDF5TypeClass.Integer && type.unsigned ? 'unsigned' : ''
-    } ${size}-bit`;
-
-    if (endianness === 'Not applicable') {
-      return typeInfos;
-    }
+    const endiannessLabel = ENDIANNESS_LABELS[endianness];
 
-    return `${typeInfos}, ${ENDIANNESS_LABELS[endianness]}`;
+    return `${type.class}, ${size}-bit${
```

Comment:
`HDF5TypeClass.Unsigned` is `"unsigned integer"`, so we no longer need the extra ternary here.